### ISSUE TITLE
No option to close the project

### ISSUE THREAD

**Issue by adityaSplendornet** [taigaio/taiga-front#452]
There is currently no option to close the project once all the activities are done with it. Can we add something to accomplish this? Also if this is added, we will need facility to show/list closed projects from admin side to reopen if needed.

**Comment by superalex** [comment]
Hello @adityaSplendornet,

Actually there is no way of closing/archiving projects but it's something that could be very useful. I think it would be better to close/archive them explicitly. We could ignore closed/archived projects in the app except in the project listing page (we are creating a new one) where could have a filter for closed/archived and allow reactivate them

**Comment by adityaSplendornet** [comment]
I agree @superalex , it is definitely useful, as the project list grows longer and longer, this feature definitely would be missed.

**Comment by superalex** [comment]
Yes!, it has been in our roadmap for a long time we just need to find time for working on it (or find someone with time) :+1:

**Comment by adityaSplendornet** [comment]
Let me see if I can get someone from my team to work on it, can you please create a branch for this work and let me know? Or do you want me to create one?

**Comment by superalex** [comment]
Thank you very much @adityaSplendornet!, 

You could fork the repo, work on your fork and then ask for a PR in the master branch.

Regards!,

**Comment by willianpaixao** [comment]
@superalex and @Xaviju,
what's the current status? If anyone is working on it, I can do it.

**Comment by bameda** [comment]
Nobody is working on it.

**Comment by marbetschar** [comment]
any chance we can get this done?

**Comment by themightychris** [comment]
@superalex @Xaviju could someone provide some implementation notes on this feature? I could really use it and might have a go at it. Seems like the required steps would be along the lines of:

1. Add a "Status" field to the Project model with values `Open` / `Closed` (or `Current`/`Archived`)
2. Create migration script to add column to existing databases with default value of `Open` (or `Current`)
3. Have the projects page and projects menu filter to only open/current projects by default
4. Add a link/button to the projects page that switches to showing closed/archived projects via a query string param or subroute

Anyone have thoughts on preferred terminology?

**Comment by goetzk** [comment]
@themightychris If you're sure there will only be two states, a bool probably makes more sense. My concern with recommending that is once the functionality is in place its likely more states will be requested...

**Comment by themightychris** [comment]
@goetzk good point, and on that note I'm leaning towards the terminology being `archived` as this is a UI-centric feature and "archived" sounds more like a UI preference vs "closed" which I feel implies something deeper about the status of a project.

**Comment by goetzk** [comment]
I don't have a strong suggestion either way, other than to suggest looking at which terms are already in use by Taiga and possibly similar tools.

**Comment by pcorbett** [comment]
+1 for "Archived." This terminology is used regularly in other platforms we use (Redbooth, Harvest, Jira).

**Comment by entdevs** [comment]
It's been awhile since anyone has discussed this enhancement. Is it being worked on?

**Comment by luisogandob** [comment]
+1 for this feature.

**Comment by entdevs** [comment]
Still waiting for an update.

**Comment by jmhlawatsch** [comment]
+1 for this feature.

**Comment by Doctor-Who** [comment]
+1 for this feature and we can fund it if need

**Comment by NatachaMenjibar** [comment]
We have been doing some housekeeping work prior to Taiga6 upcoming release and we are reviewing old issues that we didn't correctly attend back then. 
Taiga6 brings a significant UX/UI rework, and after the launch, we’ll come back to this issue as it’s part of our roadmap. This time we will make sure to review it properly. 

Thank you very much on behalf of the Taiga Team!

**Comment by alexander-potemkin** [comment]
Thank you, looking forward for that! And thank you for the UX/UI rework - Taiga is a very nifty now!

**Comment by nuokh** [comment]
> We have been doing some housekeeping work prior to Taiga6 upcoming release and we are reviewing old issues that we didn't correctly attend back then.
> Taiga6 brings a significant UX/UI rework, and after the launch, we’ll come back to this issue as it’s part of our roadmap. This time we will make sure to review it properly.
> 
> Thank you very much on behalf of the Taiga Team!

Thanks for keeping us updated! Currently testing Taiga6 in an on prem docker container and love it so far. Looking forward to this feature!

**Comment by SplendornetTech** [comment]
Looking forward to the new release and looking forward for this feature

**Comment by Link2Twenty** [comment]
+1

**Comment by softwaremagico** [comment]
I am interested on this feature also!

**Comment by fouram** [comment]
+1
